app/src/main/java/com/github/se/eventradar/ui/hosting/CreateEvent.kt

```diff
@@ -0,0 +1,299 @@
+                        onValueChange = viewModel::onEndTimeChanged,
+                        errorState = uiState.endTimeIsError)
+                  }
+              LocationDropDownMenu(
+                  value = uiState.location,
+                  onLocationChanged = viewModel::onLocationChanged,
+                  label = { Text("Location") },
+                  modifier =
+                      Modifier.fillMaxWidth()
+                          .padding(start = 35.dp, top = 8.dp, end = 35.dp)
+                          .testTag("locationDropDownMenuTextField"),
+                  getLocations = viewModel::updateListOfLocations,
+                  locationList = uiState.listOfLocations)
+
+              DropdownInputTextField(
+                  modifier =
+                      Modifier.fillMaxWidth()
+                          .padding(start = 35.dp, top = 8.dp, end = 35.dp)
+                          .testTag("ticketNameDropDownMenuTextField"),
+                  textFieldLabel = "Ticket Name",
+                  textFieldValue = uiState.ticketName,
+                  onValueChange = viewModel::onTicketNameChanged,
+                  errorState = uiState.ticketNameIsError,
+                  options = listOf("Standard", "VIP"),
+                  toggleIconTestTag = "ticketNameToggleIcon") // temporary
+
+              StandardInputTextField(
+                  modifier =
+                      Modifier.fillMaxWidth()
+                          .padding(start = 35.dp, top = 8.dp, end = 35.dp)
+                          .testTag("ticketQuantityTextField"),
+                  textFieldLabel = "Ticket Quantity",
+                  textFieldValue = uiState.ticketCapacity,
+                  onValueChange = viewModel::onTicketCapacityChanged,
+                  errorState = uiState.ticketCapacityIsError,
+              )
+
+              StandardInputTextField(
+                  modifier =
+                      Modifier.fillMaxWidth()
+                          .padding(start = 35.dp, top = 8.dp, end = 35.dp)
+                          .testTag("ticketPriceTextField"),
+                  textFieldLabel = "Ticket Price",
+                  textFieldValue = uiState.ticketPrice,
+                  onValueChange = viewModel::onTicketPriceChanged,
+                  errorState = uiState.ticketPriceIsError,
+              )
+              MultiSelectDropDownMenu(
+                  value = "Organiser",
+                  onSelectedListChanged = viewModel::onOrganiserListChanged,
+                  label = { Text("Organisers") },
+                  modifier =
+                      Modifier.fillMaxWidth()
+                          .padding(start = 35.dp, top = 8.dp, end = 35.dp)
+                          .testTag("organisersMultiDropDownMenuTextField"),
+                  getFriends = viewModel::getHostFriendList,
+                  friendsList = uiState.hostFriendsList)
+              // add a Button!
```

Comment:
?

Reply:
sorry this was just me telling myself things while I was coding, will remove